lib/http.c

```diff
@@ -3895,7 +3893,7 @@ static CURLcode http_rw_hd(struct Curl_easy *data,
                 k->httpcode = (p[0] - '0') * 100 + (p[1] - '0') * 10 +
                   (p[2] - '0');
                 p += 3;
-                if(ISSPACE(*p))
+                if(ISBLANK(*p))
```

Comment:
@badger Just FYI, the change on this line 3896, which looks fine regarding compliance with the HTTP spec, caused test failures in GDAL regression test suite where we generated at hand an invalid HTTP response header line that was just "HTTP/1.1 200\r\n", missing the space after the response code and between \r. Cf https://github.com/OSGeo/gdal/commit/bb4e0ed67154d70ab17fad6752f23af6e161bc56

Reply:
liboauth2 has the same thing.

The change above was a bugfix, as I believe the code actually *tried* to do that check already but because it was using ISSPACE instead of ISBLANK it (wrongly) happily accepted a newline where a space is mandated by the spec.

Something tells me this particular change might see further ripple effects.

Reply:
Fixed by 5b6d3291 #16728